packages/client/hmi-client/src/components/home/tera-project-table.vue

```diff
@@ -92,18 +140,46 @@ function formatStatTooltip(stat, displayName) {
 	return `${stat} ${displayName}${stat === 1 ? '' : 's'}`;
 }
 
-function getColumnWidth(columnField: string) {
-	switch (columnField) {
-		case 'description':
-			return 40;
-		case 'name':
-			return 40;
-		case 'score':
-			return 5;
-		default:
-			return 100;
-	}
+async function getProjectAssets(event: PageState = pageState) {
+	pageState = event; // Save the current page state so we still know it when the watch is triggered
+	if (isEmpty(props.searchQuery)) return;
+
+	const { rows, first } = event;
+	const searchQuery = props.searchQuery.toLowerCase().trim();
+
+	// Just fetch the asset data we are seeing in the current page
+	projectsWithKnnMatches.value.slice(first, first + rows).forEach(async (project) => {
+		// If assets were fetched before from when we were on that page don't redo it
+		if (!isEmpty(project.projectAssets) && prevSearchQuery === searchQuery) return;
+
+		const projectWithAssets = (await ProjectService.get(project.id)) as ProjectWithKnnSnippet | null;
+		if (!projectWithAssets) return;
+
+		project.projectAssets = projectWithAssets.projectAssets.filter(
+			(asset) => asset.assetName.toLowerCase().includes(searchQuery) && asset.assetType !== AssetType.Simulation // Simulations don't have names
```

Comment:
I don't know what you are trying to do here, but we just recently added NotebookSession as a projectAsset and it also doesn't have a name by default.

Reply:
I am filtering the assets to show below the project title